contrib/blueflood-minikube/README.md

```diff
@@ -0,0 +1,73 @@
+# Blueflood in minikube
+
+This is a basic, working Blueflood in minikube, complete with Cassandra, Elasticsearch, and Grafana dashboards. If all
+goes as planned, then as long as you've already [installed minikube](https://minikube.sigs.k8s.io/docs/start/), it's
+just a few steps to get it running.
+
+## Getting it started
+
+First, start minikube:
+
+```bash
+minikube start
+```
+                                               
+Next, you have to mount the blueflood project directory into minikube with:
+
+```bash
+minikube mount $FULL_PATH_TO_BLUEFLOOD:/blueflood
+```
+
+Then, if you want to run the latest Blueflood image from DockerHub, you can go ahead and start the stack with `minikube
+apply -f blueflood.yaml`. If you prefer to run a locally built version of Blueflood, you can build a Docker image inside
+minikube. You just need to set a couple of variables to do that. First, run `minikube docker-env`, and note the values
+of `DOCKER_HOST` and `DOCKER_CERT_PATH` for use in the following commands. Replace `tcp://` in `DOCKER_HOST` with
+`https://` when you use it below. Then you can build blueflood and run it with
+
+```bash
+mvn package docker:build -DdockerHost=$MODIFIED_DOCKER_HOST -DdockerCertPath=$DOCKER_CERT_PATH
+minikube apply -f blueflood.yaml
```

Comment:
Does this need to be `minikube kubectl -- apply -f blueflood.yaml` and run from ${projectRootDir}/blueflood/contrib/blueflood-minikube location? minicube did not recognize `apply` argument. Perhaps there was an alias? Also does the command need to specify path of blueflood.yaml?

Reply:
No you're right. That's supposed to be `kubectl`. [Fixed](https://github.com/rax-maas/blueflood/compare/5cf19a1c1ce206b8a5ab0a6f0fbdf28eb43d3f8b..01933f15e8b7e99088d344516c92e2fa745cbcc4?diff=unified). I also clarified the expected paths.